Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileSyncConnection/MSFT_xSPUserProfileSyncConnection.psm1

```diff
@@ -0,0 +1,231 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [parameter(Mandatory = $true)]  [System.String] $Name,
+        [parameter(Mandatory = $true)]  [System.String] $Forest,
+        [parameter(Mandatory = $true)]  [System.Management.Automation.PSCredential] $ConnectionCredentials,
+        [parameter(Mandatory = $true)] [System.String] $UserProfileService,
+        [parameter(Mandatory = $true)] [System.String[]] $IncludedOUs,
+        [parameter(Mandatory = $false)] [System.String[]] $ExcludedOUs,
+        [parameter(Mandatory = $false)] [System.String] $Server,
+        [parameter(Mandatory = $false)] [System.String] $Force,
+        [parameter(Mandatory = $false)] [System.Boolean] $UseSSL,
+        [parameter(Mandatory = $false)] [System.String] $ConnectionType,
```

Comment:
Looks like you specify the ValueMap in the schema for this, so you need the correspinding value map here as well. The recently fixed MOF schema tests will pick this up on the next test run too.